packages/components/src/Menu/Menu.tsx

```diff
@@ -284,12 +283,12 @@ function Action({
       onClick={onClick}
       ref={actionButtonRef}
     >
-      {icon && (
-        <span className={styles.icon}>
-          <Icon name={icon} />
-        </span>
-      )}
-      <Typography element="span" size="base" textColor="text">
+      {icon && <Icon name={icon} color="interactiveSubtle" />}
```

Comment:
Do we want this color here?  What about those[ instances](https://jobber.atlassian.net/browse/JOB-87494) where we want the icon color to come through? With this set here, we won't be able to see those different icon colors in the menu.

Reply:
Matt actually split that out into a separate ticket but I might as well do it here!

Reply:
![image](https://github.com/GetJobber/atlantis/assets/39704901/e6368298-f0e7-4471-b456-f5fbb1c0dafe)

Reply:
https://github.com/GetJobber/atlantis/pull/1840/commits/cb6de7f15a683f2ffa70924916ea3f3b30c44aaf

Reply:
Brilliant! Thank you for doing that here